sdk/go/common/workspace/project.go

```diff
@@ -102,6 +109,8 @@ type Project struct {
 
 	// Options is an optional set of project options
 	Options *ProjectOptions `json:"options,omitempty" yaml:"options,omitempty"`
+
+	Providers []*ProviderOptions `json:"providers,omitempty" yaml:"providers,omitempty"`
```

Comment:
No need for pointer here, we don't expect anything to be nil in this array.
```suggestion
	Providers []ProviderOptions `json:"providers,omitempty" yaml:"providers,omitempty"`
```